wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,427 @@
+
+| Artist                                   | Status                               | Exceptions                                                  | Notes                                                                                                               |
+| ---------------------------------------- | ------------------------------------ | ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| -45                              ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/297)  |
+| :Poin7less                       ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/275)  |
+| *namirin                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/18)   |
+| 2ToneDisco                       ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/157)  |
+| 3R2 / DJ Mashiro                 ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/187)  |
+| 40mP                                     | 🔴 **Disallowed**                   |                                                             |                                                                                                                     |
+| 69 de 74                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/268)  |
+| 7_7                              ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/356)  |
+| a_hisa                           ![](FA) | 🟠 **Requires explicit permission** | Before uploading beatmaps, obtain the artist's permission.  | Artist can be reached via [email](mailto:hisaweb_info@yahoo.co.jp) or via [Bandcamp](https://a-hisa.bandcamp.com/). |
```

Comment:
if you take my earlier suggestion about removing all the FA notes, then the "Exceptions" and "Notes" columns can probably just be combined to compact rows like this one.